Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+
+  <PropertyGroup>
+    <Product>YAXLib</Product>
+    <Authors>Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Authors>
```

Comment:
```suggestion
    <Authors>YAXLib maintainers and other contributors</Authors>
```

Reply:
This is possible, but I think this should not be in this PR (scope creep IMO)

Reply:
Agree.